client/src/panels/panel_declarations/covid/covid_estimates.js

```diff
@@ -512,6 +340,209 @@ class CovidEstimatesPanel extends React.Component {
       })
     );
   }
+
+  SummaryTab = ({ args: panel_args, data }) => {
```

Comment:
I find it is a little awkward that `tab_content_configs` and `SummaryTab` are properties of `CovidEstimatesPanel` now, especially since the other tab components are not. I think this is something of a code smell to look out for, when you have to move big blocks of code around to change their scope access. Usually going to be a better way.

It's pretty common to need a child component to have both read and write access to some state from some parent component. The standard way to do this is to pass down the state alongside some sort of setter callback. Some benefits:
  - the callback limits the child component's control of the parent's state, can enforce validation etc
  - keeps components composable
    - parent and child only interact through the child's API. Either could potentially be used with any other parents or children that implement an equivalent API
    
The pattern is in use here, you can see it with the `YearSelectionTabs` component and `CovidEstimatesPanel`'s `on_select_year` method.